pkgs/servers/mail/public-inbox/default.nix

```diff
@@ -137,12 +149,14 @@ buildPerlPackage rec {
   installTargets = [ "install" ];
   postInstall = ''
     for prog in $out/bin/*; do
-        wrapProgram $prog --prefix PATH : ${lib.makeBinPath [
-          git
-          /* for InlineC */
-          gnumake
-          stdenv.cc.cc
-        ]}
+        wrapProgram $prog \
+            --set NIX_CFLAGS_COMPILE_${stdenv.cc.suffixSalt} -I${lib.getDev libxcrypt}/include \
+            --prefix PATH : ${lib.makeBinPath [
```

Comment:
Those should be intended with 2 spaces

Reply:
The manual says

> Use 2 spaces of indentation per indentation level in Nix expressions, 4 spaces in shell scripts. 

My interpretation of this has always been that it applies based on language, not file extension.  Here, we're in shell code.

Reply:
nix files use 2 spaces for shell code, too. Pure shell scripts use 4 spaces.

Reply:
I don't know what to tell you apart from that I disagree (it doesn't make any sense to format shell code differently depending on whether it's in a Nix string vs a file of its own), and the documentation doesn't clarify.  We have plenty of examples of both.

Reply:
To my knowledge that is what we are currently doing. My solution would be to just format everything with 2 spaces. 🤷🏼

Reply:
We have both styles in Nixpkgs.  (I've been writing Nix expressions this way for five years, and I learned to do it this way because this was how whatever code I was looking at at the time did it.)